app/views/account/tasks/show.html.slim

```diff
@@ -1,9 +1,9 @@
 .container
   .content
-    h3 Title
-    = @task.title
+    .title
+      = title "#{@project.name}: #{@task.title}"
     h3 Description
-    = @task.description
+    = description @task.description
```

Comment:
SAME HERE

for description too, don't mix

such values COULD AND WILL BE different